simple/cq_data.c

```diff
@@ -99,7 +99,7 @@ static int alloc_ep_res(struct fi_info *fi)
 
 	memset(&cq_attr, 0, sizeof cq_attr);
 	cq_attr.format = FI_CQ_FORMAT_DATA;
-	cq_attr.wait_obj = FI_WAIT_NONE;
+	cq_attr.wait_obj = FI_WAIT_FD;
```

Comment:
There isn't any specific reason. I can update it to FI_WAIT_UNSPEC since we are not retrieving the FD in this example.